src/abstractdataframe/reshape.jl

```diff
@@ -1,12 +1,13 @@
 """
     stack(df::AbstractDataFrame, [measure_vars], [id_vars];
-          variable_name::Symbol=:variable, value_name::Symbol=:value, view::Bool=false)
+          variable_name::Symbol=:variable, value_name::Symbol=:value,
+          view::Bool=false, vareltype::Type=CategoricalString)
```

Comment:
Maybe `variable_eltype` for consistency with `variable_name`?

Reply:
good point. Fixed